webapp/components/admin_console/purge_caches.jsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import Client from 'client/web_client.jsx';
+import * as Utils from 'utils/utils.jsx';
+
+import {FormattedMessage, FormattedHTMLMessage} from 'react-intl';
+
+export default class PurgeCachesButton extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.handlePurge = this.handlePurge.bind(this);
+
+        this.state = {
+            loading: false,
+            fail: null
+        };
+    }
+
+    handlePurge(e) {
+        e.preventDefault();
+
+        this.setState({
+            loading: true,
+            fail: null
+        });
+
+        Client.invalidateAllCaches(
+            () => {
+                this.setState({
+                    loading: false
+                });
+            },
+            (err) => {
+                this.setState({
+                    loading: false,
+                    fail: err.message + ' - ' + err.detailed_error
+                });
+            }
+        );
+    }
+
+    render() {
+        if (global.window.mm_license.IsLicensed !== 'true') {
+            return <div/>;
+        }
+
+        let testMessage = null;
+        if (this.state.fail) {
+            testMessage = (
+                <div className='alert alert-warning'>
+                    <i className='fa fa-warning'/>
+                    <FormattedMessage
+                        id='admin.purge.purgeFail'
+                        defaultMessage='Purging unsuccessful: {error}'
+                        values={{
+                            error: this.state.fail
+                        }}
+                    />
+                </div>
+            );
+        }
+
+        const helpText = (
+            <FormattedHTMLMessage
```

Comment:
can you make this `<FormattedMessage>` you also ain't using html tags in the string, so no need to use this one here, also we should try and avoid using `FormattedHTMLMessage`